### PR TITLE
LDP-888: Update composer  drupal core dependency.

### PR DESCRIPTION
**Pull request by vasike**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "psr-4": { "drunomics\\ServiceUtils\\": "src/" }
   },
   "require": {
-    "drupal/core": "8.9.*"
+    "drupal/core": "^8.9 || ^9"
```
Comment (fago) [inline composer.json]: read the README about branching strategy & releases. let's keep following that. #nw

**Review by fago — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (vasike) [reply composer.json]: @fago so do we need a new branch?

> Major version numbers match Drupal 8 core compatibility, e.g.

So what should we have for Drupal 9?

Comment (mostepaniukvm) [reply composer.json]: @vasike 
I think it should be obvious what branch should be for d9.2
https://github.com/drunomics/service-utils/tree/9.1.x#versions

Comment (vasike) [reply composer.json]: @mostepaniukvm  / @fago updated the base branch and also the drupal version

the question is 9.1.x is the right one or do we need a new one ... 9.2.x ?

**Comment by mostepaniukvm** [comment]
created 9.2.x branch instead and no need to merge it

**Pull request closed without merging**